nvim/lua/env/remaps.lua

```diff
@@ -0,0 +1,20 @@
+vim.g.mapleader = " "
+
+vim.keymap.set("n", "<leader>n", ":NERDTreeToggle<CR>")
+vim.keymap.set("n", "<leader>m", ":TagbarToggle<CR>")
+vim.keymap.set("n", "<leader>no", ":nohlsearch<CR>")
+
+vim.keymap.set("n", "<leader>v", "<C-w>v<C-w>")
+vim.keymap.set("n", "<leader>r", ":res +10<CR>")
+vim.keymap.set("n", "<leader>R", ":res -10<CR>")
+vim.keymap.set("n", "<leader>vr", ":vert res +5<CR>")
+vim.keymap.set("n", "<leader>vR", ":vert res -5<CR>")
+
+vim.keymap.set("n", "<leader>c", ":%s/\\<<C-r><C-w>\\>/<C-r><C-w>/gI<Left><Left><Left>")
```

Comment:
Might be a good idea to comment what this does. I have already forgotten it.

Reply:
Comment added.